static/schemas/obms/redfish-obm-service.json

```diff
@@ -0,0 +1,57 @@
+{
+    "title": "redfish-obm-service",
+    "definitions": {
+        "Obm": {
+            "description": "Redfish OBM settings",
+            "type": "object",
+            "properties": {
+                "nodeId": {
+                    "type": "string"
+                },
+                "service": {
+                    "type": "string"
+                },
+                "config": {
+                    "type": "object",
+                    "properties": {
```

Comment:
Suggest adding an "additionalProperties: false" in order to prevent those extra and useless properties from being added. And also, if we add new setting in future, this will catch the error if developer forgots the schema definition of new setting